pkg/authentication/handlers.go

```diff
@@ -73,6 +73,7 @@ type API struct {
 	oauth2           OAuth2ContextInterface
 	helper           OAuth2HelperInterface
 	cookieManager    AuthCookieManagerInterface
+	identities       SessionManagerInterface
```

Comment:
naming seems a bit off for now?maybe sessions or sessionManager?

```suggestion
	sessions       SessionManagerInterface
```